packages/block-library/src/site-logo/edit.js

```diff
@@ -87,7 +86,11 @@ const SiteLogo = ( {
 		);
 		return {
 			title: siteEntities?.name,
-			...pick( getSettings(), [ 'imageEditing', 'maxWidth' ] ),
+			...Object.fromEntries(
+				Object.entries( getSettings() ).filter( ( [ key ] ) =>
```

Comment:
Will always be an object, no need to add a fallback:

https://github.com/WordPress/gutenberg/blob/a4040e40965b5430874413e5d5389ac889ad13b9/packages/block-editor/src/store/defaults.js#L36